terraform/meshdb_cluster/output.tf

```diff
@@ -0,0 +1,25 @@
+output "mgr_ip" {
+  description = "IP address of the managment node"
+  value = {
+    for k, mgr in proxmox_vm_qemu.meshdbmgr : k => mgr.default_ipv4_address
+  }
+}
+
+output "worker_ips" {
+  description = "IP address of the worker node"
+  value = {
+    for k, node in proxmox_vm_qemu.meshdbnode : k => node.default_ipv4_address
+  }
+}
+
+output "lb_ip" {
+  description = "IP address of the lb"
+  value = {
+    "0" = var.meshdb_lb_ip
+  }
+}
+
+#resource "local_file" "kubeconfig" {
+#  content  = module.k3s.kube_config
+#  filename = "${path.module}/k3s.${var.meshdb_env_name}.yaml"
+#}
```

Comment:
Where is the kubeconfig going to live now?

Reply:
On the mgr box, accessed via ssh

Reply:
I'd prefer if the kubernetes nodes themselves did not have any kind of persistent config/command line tools. It would be better to store it on the `lb` node.

Reply:
I don't know what would happen if I delete it. It is setup by k3s not me.